templates/firefox-ios.xliff

```diff
@@ -3649,6 +3701,10 @@ Private Tab</source>
         <source>When you’re signed in and synced, you’re safer. %@ encrypts your passwords, bookmarks, and more.</source>
         <note>String used to describes the description of what Firefox is on the Sync onboarding page for current version in our Onboarding screens. Placeholder is for the app name.</note>
       </trans-unit>
+      <trans-unit id="Onboarding.Sync.Description.v123" xml:space="preserve">
+        <source>%@ encrypts your passwords, bookmarks, and more when you're synced.</source>
```

Comment:
Needs proper apostrophe

Reply:
I... can't tell? What apostrophe here?

Reply:
you're -> you’re 
(you can see in this PR that the linter returns errors because of the three apostrophes)

Reply:
Documentation on typography [here](https://mozilla-l10n.github.io/documentation/misc/documentation_styleguide.html?highlight=apostrope#typography) calls out use of typographical apostrophes

Reply:
Link to Firefox Content styleguide: https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o